appveyor.yml

```diff
@@ -50,5 +50,8 @@ test_script:
   - codecov -f profile.cov -F windows
 
 # uncomment to debug builds
-# on_finish:
+#on_finish:
+#  - set | perl -pe "s!^!set !g" > C:\Users\appveyor\Desktop\runme.bat
+#  - echo cd %APPVEYOR_BUILD_FOLDER% >> C:\Users\appveyor\Desktop\runme.bat
+#  - echo cmd >> C:\Users\appveyor\Desktop\runme.bat
 #  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```

Comment:
if you wanted, you could powershell it instead of perl. 
`
get-childitem env: |foreach-object {Out-File -append -FilePath c:\users\appveyor\Desktop\runme.bat -InputObject "set $($_.Name)=`"$($_.Value)`""}
`

Reply:
Thanks Derek, as discuss offline appveyor/yaml make this harder that it should be `appveyor error: Mapping values are not allowed in this context.`
So let's keep the gnu perl as gnu tools are available on the path